shared/src/common/types/eam-group/eam-group-get-response-dto.type.ts

```diff
@@ -0,0 +1,9 @@
+type EAMGroupGetResponseDto = {
+  id: string;
+  name: string;
+  createdAt: string;
+  users: { id: string; name: string }[] | [];
+  permissions: { id: string; name: string }[];
```

Comment:
```suggestion
  users: { id: string; name: string }[];
  permissions: { id: string; name: string }[];
```
create a type for it. Or I think we can reuse user & permission DTOs